src/UI/Buyer/src/app/shared/components/template-dropdown/template-dropdown.component.html

```diff
@@ -0,0 +1,29 @@
+<div (click)="toggleItems()"
```

Comment:
I think  you made something really cool, and I will accept this PR with no revision for this section  *but* I think the whole drop-down thing suffers from a couple of issues that I've outline here - https://github.com/ordercloud-api/ngx-shopper/issues/63 and I really should've talked to you before you started working on this component.

Mainly, this doesn't work for anyone who has a large number of spending accounts, which in my experience happens fairly often. 

I think this is a good opportunity to make a generic version of what that I had outlined in the linked issue. A searchable, paginated, selectable list of ANYTHING. That one is specific to addresses but it occurs to me now that in the buyer app alone Addressess, SpendingAccounts, and CreditCards could benefit from this pattern.